src/index.js

```diff
@@ -82,7 +82,7 @@ const useKrpano = (options = {}) => {
   // Set container div `id` when ref is assigned
   useEffect(() => {
     if (containerRef.current) {
-      containerRef.current.id = DEFAULT_TARGET_ID
+      containerRef.current.id = embeddingParams.target || DEFAULT_TARGET_ID
```

Comment:
This topic is literally missing from the library, so thank you for bringing it up!
I was thinking that we should have a unique`id` for each container. It would be better to generate a random ID for them. Would you mind updating the implementation? If not, I can do it this weekend.

Reply:
Done